Pre-built Linux binary exists in tarball

--- origsrc/free42/gtk/Makefile	2012-10-27 14:14:48.000000000 -0500
+++ src/free42/gtk/Makefile	2012-11-11 15:46:34.289801700 -0600
@@ -67,6 +67,7 @@ $(EXE): $(OBJS)
 	$(CXX) -o $(EXE) $(LDFLAGS) $(OBJS) $(LIBS)
 ifdef OLPC
 	tar xvfz Free42.activity.tgz
+	rm -f Free42.activity/bin/*
 	cp $(EXE) Free42.activity/bin
 	zip -r Free42.xo Free42.activity
 	rm -rf Free42.activity
