--- origsrc/free42-nologo-1.5.7/gtk/Makefile	2016-02-28 13:14:13.000000000 -0600
+++ src/free42-nologo-1.5.7/gtk/Makefile	2016-03-29 14:09:35.039625200 -0500
@@ -15,7 +15,7 @@
 # along with this program; if not, see http://www.gnu.org/licenses/.
 ###############################################################################
 
-CXXFLAGS := -MMD \
+CXXFLAGS += -MMD \
 	    -Wall \
 	    -Wno-parentheses \
 	    -Wno-write-strings \
